db/migrate/20150616140258_create_db_activities.rb

```diff
@@ -0,0 +1,18 @@
+class CreateDbActivities < ActiveRecord::Migration
+  def change
+    create_table :db_activities do |t|
+      t.integer :user_id, null: false
+      t.integer :recipient_id
+      t.string  :recipient_type
+      t.integer :trackable_id, null: false
+      t.string  :trackable_type, null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.